packages/frontend/src/components/header/ProjectSummary.tsx

```diff
@@ -19,7 +19,7 @@ export interface ProjectSummaryStat {
 }
 
 export function ProjectSummary(props: Props) {
-  const cols = props.type === 'bridge' ? 4 : 3
+  const cols = props.type === 'layer2' ? 3 : 4
```

Comment:
maybe add this value to props, or write switch case with non exhaustive check. I am worried that this ternary operator will be hard to maintain in the future